operator/Makefile

```diff
@@ -65,11 +66,21 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build -t ${IMG}:latest .
+	docker tag ${IMG}:latest ${IMG}:${COMMIT_HASH}
+	docker tag ${IMG}:latest ${IMG}:dev
 
 docker-push: ## Push docker image with the manager.
-	docker tag ${IMG} localhost:37129/${IMG}
-	docker push localhost:37129/${IMG}
+	docker push ${IMG}:latest
+
+helm-build: manifests ## Build helm chart with the manager.
+	cp config/crd/bases/* ../charts/logging-pipeline-plumber/crds/
```

Comment:
can we rename this to generate? that way we can put any generation commands under it.

Reply:
will fix it with makefile changes